config/crd/bases/infrastructure.cluster.x-k8s.io_gcpmachines.yaml

```diff
@@ -101,6 +101,11 @@ spec:
                   Defaults to 30.
                 format: int64
                 type: integer
+              rootDeviceType:
+                description: 'RootDeviceType is the type of the root volume. Supported
+                  types of root volumes: 1. "pd-standard" - Standrad (HDD) persistent
```

Comment:
Still a typo here and in config/crd/bases/infrastructure.cluster.x-k8s.io_gcpmachinetemplates.yaml

Reply:
Doh, forgot to rerun `make generate`. Fixed (looks like it also picked up unrelated doc string change that was made previously.